### PR TITLE
add missing Exprs in Walk

### PR DESCRIPTION
**Pull request by tbg**
(go-fuzz)

Should probably have a test that makes sure `WalkExpr` can handle all implementers of `Expr` at all times.

### CODE REVIEW

File: sql/parser/walk.go [hunk 1]
```diff
@@ -86,6 +86,11 @@ func WalkExpr(v Visitor, expr Expr) Expr {
 	case *QualifiedName:
 		// Terminal node: nothing to do.
 
+	case Row:
```
Comment (tamird) [inline sql/parser/walk.go]: should this be a cast to Tuple outside the switch?

Comment (tschottdorf) [inline sql/parser/walk.go]: I don't think so. That runs the risk of conversion where you don't want it.

Comment (petermattis) [inline sql/parser/walk.go]: `TestFillArgs` tried to be exhaustive of the expression nodes. It failed when a new one was added.

In the case of `Row`, I think it makes sense to add a `NormalizeExpr` walk that transforms `Row` into `Tuple` before `FillArgs` and `EvalExpr` are called. `Row` and `Tuple` are synonymous for execution and only differ in the SQL they generate. There are a variety of other normalizations we'll want to perform as well, such as pre-evaluating branches of the expression tree which are constant.

Comment (petermattis) [inline sql/parser/walk.go]: I'll file an issue about the above. In the meantime, add a test case to `TestFillArgs` which uses an explicit row: `ROW($1, $2, $3)`.

Comment (tschottdorf) [inline sql/parser/walk.go]: done.

**Comment by petermattis** [comment]
LGTM

**Pull request merged**